src/output/tree.rs

```diff
@@ -840,6 +837,11 @@ impl<'module, W: io::Write> Context<'module, W> {
     }
 }
 
+#[inline]
+fn name_is_ascii_string(name: &str) -> bool {
+    name.contains("ascii") && name.contains("string")
+}
+
```

Comment:
Not sure about this rule :thinking:

Reply:
It is the cleanest way I saw to get bespoke ascii-strings working as ascii string. Consider `tar.rs` `tar.ascii-string(.opt0(.nonempty)?)?` How should we handle those without adding a new override for every format?

Reply:
Alternatively, we could have an `AsAsciiString` format wrapper that makes whatever s inside of it treated as though ascii, but that could get us in trouble due to weird constructions.

Reply:
It could be narrowed to `name.contains("asciiz-string") || name.contains("ascii-string")` if you like...

Reply:
Which would satisfy all the current passing cases while filtering out a whole class of potential false positives down the line.

Reply:
Perhaps using `ascii-char` would be more flexible, but perhaps we would need `ascii-char-non-nul`?

Reply:
Probably so.

Reply:
Possibly also an explicit `ascii-char.non-null` and `ascii-char.null` to keep the same prefix match for the ItemVar names.

Reply:
Relegated to issue #88